### DISCUSSION TITLE
Modifying disk SSD and HDD properties during use

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#34412] [A - General / Q&A]
version:2.0.2
I did not consider the issue of hot and cold separation when creating the cluster. Now, I would like to enable this strategy. Is there a way to modify the disk SSD or HDD without recreating the cluster? Currently, I am using the default unspecified HDD.
I have tried to stop the node and modify storage_root_path=/data1/doris/be/storage to storage_root_path=/data1/doris/be/storage, media: HDD. It is possible to create a hot partition table, but the data was not transferred to the cold storage disk after it was cooled down

**Comment by yagagagaga** [comment]
You need to add at least 2 disks in your storage_root_path. One for SSD and another for SSD, which can make your hot/cold separation work.

```properties
storage_root_path=/data1/doris/be/storage, media: HDD;/data2/doris/be/storage, media: SSD
```

**Reply by fdf1779** [reply]
Can I understand that SSD and HDD can only take effect on the same node and cannot take effect across nodes

**Reply by fdf1779** [reply]
I now want different nodes in a cluster to have different storage disks, such as Node1 being all SSDs and Node2 being all HDDs, for cold and hot layering
